use-cases/azure.md

```diff
@@ -0,0 +1,319 @@
+
+In the new left-hand menu, select **Manage -> All applications**. Use the **Search Bar** and/or the available filters to find your application.
+
+![Azure AD Enterprise Applications](../assets/azure-ad-enterprise-application.png)
+
+Click on the application you just created, for which you want to assign roles to users/groups, then, in the left-hand navigation menu, Select **Manage -> Users and groups**.
+
+Click on **Add user/group** to open the **Add Assignment** pane.
+
+Below **Users**, click on *None Selected* and, on the **Users** pane that has just opened on the right, search and select the users/groups you want to assign roles to.
+
+Once you've selected users and groups, click on the **Select** button.
+
+Back to the **Add assignment** pane, below **Select a Role**, click on *None Selected* and, on the **Select a role** pane that has just opened on the right, search and select the role you want to assign to the selected users.
+
+> :bulb: If only one role is available for your application, it would be automatically selected and greyed by default
+
+Choose a role (only a single role can be selected at a time), click on the **Select** button, and click on the **Assign** button to finalize the assignment of users and groups to the app.
+
+:repeat: Repeat the operations for all the roles you want to assign.
+
+## Configure RabbitMQ to use Azure AD as OAuth 2.0 authentication backend
+
+The configuration on Azure side is done. You now have to configure RabbitMQ to use the resources you just created.
+
+[rabbitmq.config](../conf/azure/rabbitmq.config) is a sample RabbitMQ advanced configuration to **enable Azure AD as OAuth 2.0 authentication backend** for the RabbitMQ Management UI.
+
+Update it with the following values (you should have noted these in the previous steps):
+- **Tenant ID** associated to the app that we registered in Azure AD
+- **Application ID** associated to the app that we registered in Azure AD
+- Value of the **secret** we created for our app in Azure AD
+- Value of the **jwks_uri** key from `https://login.microsoftonline.com/{TENANT_ID}/v2.0/.well-known/openid-configuration`
+
+```
+$ vi rabbitmq.config
+[
+  {rabbit, [
+   {auth_backends, [rabbit_auth_backend_oauth2, rabbit_auth_backend_internal]}
+  ]},
+  {rabbitmq_management, [
+     {oauth_enable, true},
+     {oauth_client_id, "PUT YOUR AZURE AD APPLICATION ID"},
+     {oauth_client_secret, "PUT YOUR AZURE AD APPLICATION SECRET"},
+     {oauth_provider_url, "https://login.microsoftonline.com/AZURE_AD_TENANT_ID"}
+ ]},
+ {rabbitmq_auth_backend_oauth2, [
+   {resource_server_id, <<"PUT YOUR AZURE AD APPLICATION ID">>},
+   {extra_scopes_source, <<"roles">>},
+   {key_config, [
+     {jwks_url, <<"PUT YOUR AZURE AD JWKS URI VALUE">>}
+   ]}
+ ]}
+].
+```
+
+> :arrow_right: You will provide this file to RabbitMQ when you will start it (later in this tutorial)
+
+## Start RabbitMQ
```

Comment:
Did you try the instructions on how to start RabbitMq explained [here](http://localhost:6419/use-cases/keycloak.md#start-rabbitmq)?  in particular the very last step which suggest using an already built docker image.  Did it work for you?

The plan is that once this `oidc-integration` branch merges into `v.3.10.x` branch we drop all the instructions referring to building RabbiMQ and instead we pull one of the existing docker images.

Reply:
I didn't try before but I just did: it works perfectly so let me try to adjust the tutorial to use this method instead of building rabbitmq from scratch as I did.